From 65673e4130d755b45e7aa4433e3cdc057f915c84 Mon Sep 17 00:00:00 2001
From: Kovid Goyal <kovid@kovidgoyal.net>
Date: Wed, 31 Jul 2019 15:49:22 +0530
Subject: [PATCH 29/71] Remove metadata from conversion_options API docs since
 using it to set metadata is really not a good idea

---
 src/calibre/web/feeds/news.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/calibre/web/feeds/news.py b/src/calibre/web/feeds/news.py
index 854c62ebec..2a3797577c 100644
--- a/src/calibre/web/feeds/news.py
+++ b/src/calibre/web/feeds/news.py
@@ -210,8 +210,6 @@ class BasicNewsRecipe(Recipe):
     #:
     #:   conversion_options = {
     #:     'base_font_size'   : 16,
-    #:     'tags'             : 'mytag1,mytag2',
-    #:     'title'            : 'My Title',
     #:     'linearize_tables' : True,
     #:   }
     #:
